THIRD-PARTY-NOTICES

```diff
@@ -11,6 +11,16 @@ https://github.com/SonarSource/sonarlint-visualstudio
 
 The attached notices are provided for information only.
 
+License notice for Visual Studio Image Library (used for the UI)
+----------------------------------------------------------------
+
+The Microsoft Visual Studio 2022 Image library has it's own Licence.
+You may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://download.microsoft.com/download/0/6/0/0607D8EA-9BB7-440B-A36A-A24EB8C9C67E/Visual%20Studio%202022%20Image%20Library%20EULA.rtf
```

Comment:
Maybe we should remove the empty space before the link and add https:

```suggestion
    https://download.microsoft.com/download/0/6/0/0607D8EA-9BB7-440B-A36A-A24EB8C9C67E/Visual%20Studio%202022%20Image%20Library%20EULA.rtf
```

Reply:
The space looks like a standard for this file.